serverless.yml

```diff
@@ -1,69 +1,77 @@
 service: cru-udp-aa-sync
+frameworkVersion: '3'
+configValidationMode: error
+plugins:
+  - serverless-webpack
 
 provider:
   name: aws
   stage: ${env:ENVIRONMENT}
-  runtime: nodejs10.x
   region: us-east-1
   vpc:
     securityGroupIds:
-      - ${file(${env:ECS_CONFIG}/bin/vars.yml):us-east-1.webapp_elb_sg}
-    subnetIds: ${file(${env:ECS_CONFIG}/bin/vars.yml):us-east-1.prod_apps_1_all}
+      - sg-888294ed
+    subnetIds:
+      - subnet-f2c2dfda # us-east-1a
+      - subnet-1508db62 # us-east-1b
+      - subnet-3fd94266 # us-east-1c
   stackTags:
     Name: ${env:PROJECT_NAME}-${self:custom.environmentMap.${env:ENVIRONMENT}}
     owner: udp-support@cru.org
     application: ${env:PROJECT_NAME}
     env: ${self:custom.environmentMap.${env:ENVIRONMENT}}
     managed_by: serverless-framework
     function: lambda
-  iamRoleStatements:
-    - Effect: Allow
-      Action:
-        - S3:ListBucket
-      Resource:
-        Fn::Join:
-          - ''
-          - - 'arn:aws:s3:::'
-            - ${env:S3_ECID_BUCKET}
-    - Effect: Allow
-      Action:
-        - S3:GetObject
-        - S3:PutObject
-        - S3:DeleteObject
-      Resource:
-        Fn::Join:
-          - ''
-          - - 'arn:aws:s3:::'
-            - ${env:S3_ECID_BUCKET}
-            - '/*'
-  environment: ${file(serverless/environment.js)}
-
-functions:
-  aa-sync:
-    handler: handlers/aa-sync.handler
-    timeout: 120
-    events:
-      - schedule: rate(12 hours)
+  iam:
+    role:
+      statements:
+        - Effect: Allow
+          Action:
+            - S3:ListBucket
+          Resource:
+            Fn::Join:
+              - ''
+              - - 'arn:aws:s3:::'
+                - ${env:S3_ECID_BUCKET}
+        - Effect: Allow
+          Action:
+            - S3:GetObject
+            - S3:PutObject
+            - S3:DeleteObject
+          Resource:
+            Fn::Join:
+              - ''
+              - - 'arn:aws:s3:::'
+                - ${env:S3_ECID_BUCKET}
+                - '/*'
+  layers:
+    - arn:aws:lambda:us-east-1:072686360478:layer:node-16_14_2:1
+  environment: ${file(env.js)}
 
 package:
   individually: true
-  exclude:
-    - .git/**
-    - .serverless/**
-    - .deployment/**
-
-plugins:
-  - '@cruglobal/serverless-merge-config'
-  - serverless-webpack
+  patterns:
+    - '!.webpack/**'
+    - '!test/**'
+    - '!**/*.test.js'
+    - '!.deployment/**'
 
 custom:
   webpack:
     webpackConfig: ./webpack.config.js
     includeModules: true
     packager: yarn
     excludeFiles: '**/*.test.js'
+    excludeRegex: \.ts|test|\.map
```

Comment:
Excluding typescript files? Is something else compiling them? Or perhaps there aren't any in this repo?

Reply:
No ts files in this repo. This is just the settings I've been using across all the apps. I had add ts and map to exclude to get another app under the MAX upload size. There's no need to include them in what gets sent to AWS lambda.